.github/workflows/pytest.yml

```diff
@@ -0,0 +1,194 @@
+name: Pytest
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  test:
+    strategy:
+      matrix:
+        include: #ckan-image see https://github.com/ckan/ckan-docker-base, ckan-version controls other image tags
+          - ckan-version: "2.11"
+            ckan-image: "2.11-py3.10"
+            experimental: false
+          - ckan-version: "2.10"
+            ckan-image: "2.10-py3.10"
+            experimental: false
+          - ckan-version: "master"
+            ckan-image: "master"
+            experimental: true  # master is unstable, good to know if we are compatible or not
+      fail-fast: false
+
+    name: ${{ matrix.experimental && '**Fail_Ignored** ' || '' }} CKAN ${{ matrix.ckan-version }}
+    runs-on: ubuntu-latest
+    container:
+      image: ckan/ckan-dev:${{ matrix.ckan-image }}
+      options: --user root
+    services:
+      clamav:
+        image: clamav/clamav
+        ports:
+          - 3310:3310 #clamd
+          - 7357:7357 #milter
+      solr:
+        image: ckan/ckan-solr:${{ matrix.ckan-version }}-solr9
+      postgres:
+        image: ckan/ckan-postgres-dev:${{ matrix.ckan-version }}
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        #options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+      redis:
+          image: redis:7
+    env:
+      CKAN_SQLALCHEMY_URL: postgresql://ckan_default:pass@postgres/ckan_test
+      CKAN_DATASTORE_WRITE_URL: postgresql://datastore_write:pass@postgres/datastore_test
+      CKAN_DATASTORE_READ_URL: postgresql://datastore_read:pass@postgres/datastore_test
+      CKAN_SOLR_URL: http://solr:8983/solr/ckan
+      CKAN_REDIS_URL: redis://redis:6379/1
+      CLAMAV_HOST: clamav
+      CLAMAV_PORT: 3310
+      CLAMAV_SOCKET: /tmp/clamd.sock
+
+    steps:
+    - name: Cache ClamAV signatures
+      id: cache-clamav
+      uses: actions/cache@v4
+      with:
+        path: /var/lib/clamav
+        key: clamav-signatures-${{ runner.os }}-${{ matrix.ckan-version }}-${{ github.run_id }}
+        restore-keys: |
+          clamav-signatures-${{ runner.os }}-
+
+#    - name: Cache pip
+#      uses: actions/cache@v4
+#      with:
+#        path: ~/.cache/pip
+#        key: pip-${{ runner.os }}-${{ matrix.ckan-version }}-${{ hashFiles('*requirements.txt') }}
+#        restore-keys: |
+#          pip-${{ runner.os }}-
+
+    - name: Install clamav
+      run: |
+        set -eu
+        ## For Debian Container running as root
+        
+        apt-get update
+        apt-get install -y --no-install-recommends clamav clamav-daemon
+        
+        freshclam
```

Comment:
Is `freshclam` available with the packages we have installed? On Amazon Linux we had to separately install `clamav-update`.

Reply:
it is and it works. if it breaks, we will update it.